uaclient/entitlements/fips.py

```diff
@@ -12,39 +9,7 @@ class FIPSCommonEntitlement(repo.RepoEntitlement):
                 'libssl1.0.0-hmac', 'linux-fips', 'strongswan-hmac',
                 'openssh-client', 'openssh-server', 'openssl', 'libssl1.0.0',
                 'fips-initramfs', 'strongswan']
-
-    def disable(self, silent=False, force=False):
```

Comment:
Moved ESMEntitlement.disable and FIPSEntitlement.disable to common parent RepoEntitlement.disable.